src/ontodia/data/utils.ts

```diff
@@ -0,0 +1,27 @@
+import * as N3 from "n3";
+import { RdfNode, Triple } from './sparql/sparqlModels';
+
+export function N3toRdfNode(entity: string): RdfNode {
+    if (N3.Util.isLiteral(entity)) {
+        return {type: 'literal', value: entity.substring(1, entity.length - 1), 'xml:lang': ''};
```

Comment:
`entity.substring(1, entity.length - 1)` would still produce incorrect result for
```
"42"^^<http://www.w3.org/2001/XMLSchema#integer>
```